databases/week2/part3Homework.sql

```diff
@@ -0,0 +1,23 @@
+-- Part 3: More queries
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com 
+SELECT task.id,  task.title AS task_title FROM task
+LEFT JOIN user_task ON user_task.task_id=task.id
+WHERE user_task.user_id=2;
```

Comment:
You are getting the right tasks here of course, but not really using the intended solution :smile: 

Remember that there could easily be many users with email addresses ending in "@spotify.com". Also the JOIN could just as well be an INNER JOIN in this case.

How would you come up with a more general solution?